README.md

```diff
@@ -75,6 +75,30 @@ For our need to forcibly close connections to unelected masters and handle keepe
 
 We are open to alternative solutions (PRs are welcome) like using haproxy if they can met the above requirements. For example, an hypothetical haproxy based proxy needs a way to work with changing ip addresses, get the current cluster information and being able to forcibly close a connection when an haproxy backend is marked as failed (as a note, to achieve the latter, a possible solution that needs testing will be to use the [on-marked-down shutdown-sessions](https://cbonte.github.io/haproxy-dconv/configuration-1.6.html#5.2-on-marked-down) haproxy server option).
 
+### How does Stolon differ from ____?
+
+#### Pure Kubernetes
+
+Stolon does not require a shared volume be attached to whatever node the Postgres container is running on. **Is Stolon faster than K8's pod process?? Is that another reason? I can't imagine it would be that different**. With a pure Kubernetes approach and something like an EBS volume, you can have a single Postgres master and even slave nodes with replication that could serve as read only servers. In the event of a master failure, K8S can move the DB container around and repoint the service as necessary.
```

Comment:
Around k8s and fencing I'll add that a funny point is that I stated that k8s needs fencing some time ago (https://groups.google.com/forum/#!topic/coreos-dev/ZCAywIDVyys ignore the first message...) and it was converted to a k8s issue https://github.com/kubernetes/kubernetes/issues/12338 . Now finally this has been taken care in https://github.com/kubernetes/kubernetes/pull/34160